lgn-messages/src/types/mod.rs

```diff
@@ -1,445 +0,0 @@
-use std::fmt::Display;
-use std::fmt::Formatter;
-
-use derive_debug_plus::Dbg;
-use serde_derive::Deserialize;
-use serde_derive::Serialize;
-use thiserror::Error;
-
-use crate::routing::RoutingKey;
-
-pub mod experimental;
-pub mod v1;
-
-const REQUIRED_STAKE_SMALL_USD: Stake = 98777;
-const REQUIRED_STAKE_MEDIUM_USD: Stake = 98777;
-const REQUIRED_STAKE_LARGE_USD: Stake = 169111;
-
-/// A keyed payload contains a bunch of bytes accompanied by a storage index
-pub type KeyedPayload = (String, Vec<u8>);
-
-pub trait ToKeyedPayload {
-    fn to_keyed_payload(&self) -> KeyedPayload;
-}
-
-pub type HashOutput = [u8; 32];
-
-#[derive(Clone, Debug, Deserialize, Serialize)]
-pub enum TaskType {
-    TxTrie(experimental::tx_trie::WorkerTask),
-    RecProof(experimental::rec_proof::WorkerTask),
-    V1Preprocessing(v1::preprocessing::WorkerTask),
-    V1Query(v1::query::WorkerTask),
-    V1Groth16(v1::groth16::WorkerTask),
-}
-
-#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
-pub enum ReplyType {
-    TxTrie(experimental::tx_trie::WorkerReply),
-    RecProof(experimental::rec_proof::WorkerReply),
-    V1Preprocessing(WorkerReply),
-    V1Query(WorkerReply),
-    V1Groth16(WorkerReply),
-}
-
-#[derive(Clone, PartialEq, Deserialize, Serialize)]
-pub struct MessageEnvelope<T> {
-    /// Query id is unique for each query and shared between all its tasks
-    pub query_id: String,
-
-    /// Task id is unique for each task and helps to map replies to tasks
-    pub task_id: String,
-
-    /// Task id referenced in the DB tasks table
-    pub db_task_id: Option<i32>,
-
-    /// Estimate how long it takes this task to finish.
-    /// This includes may factors like: redis queue current length, workers count, parallel queries
-    /// count, etc. Ideally assigned by an "intelligent" algorithm. Not important for now
-    /// though. Might become relevant then we have clients waiting for results, and we can
-    /// process queries relatively fast.
-    pub rtt: u64,
-
-    /// How much work prover has to do
-    pub gas: Option<u64>,
-
-    /// How and where to route the message.
-    pub routing_key: RoutingKey,
-
-    /// Details of the task to be executed.
-    pub inner: T,
-
-    /// The proving system version
-    pub version: String,
-}
```

Comment:
`rtt`, `db_task_id`, `gas`, `routing_key` were all dead code. query_id and task_id were merged